cloudinit/config/cc_phone_home.py

```diff
@@ -130,7 +130,7 @@ def handle(name, cfg, cloud, log, args):
     tries = ph_cfg.get("tries")
     try:
         tries = int(tries)  # pyright: ignore
-    except ValueError:
+    except (ValueError, TypeError):
```

Comment:
works for me. This is a better suggestion than the alternative of `tries = ph_cfg.get("tries", DEFAULT_RETRIES)`, as someone in the off chance provided `tries: null` in user-data, we'd still fail with the TypeError exception